lib/features/family/features/home_screen/cubit/family_home_screen_cubit.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:givt_app/features/children/add_member/models/member.dart';
+import 'package:givt_app/features/family/features/home_screen/presentation/models/family_home_screen.uimodel.dart';
+import 'package:givt_app/features/family/features/home_screen/usecases/family_setup_usecase.dart';
+import 'package:givt_app/features/family/features/home_screen/usecases/preferred_church_usecase.dart';
+import 'package:givt_app/features/family/features/home_screen/usecases/registration_use_case.dart';
+import 'package:givt_app/features/family/features/profiles/models/profile.dart';
+import 'package:givt_app/features/family/features/profiles/repository/profiles_repository.dart';
+import 'package:givt_app/features/family/features/reflect/presentation/models/grateful_avatar_uimodel.dart';
+import 'package:givt_app/features/impact_groups/models/impact_group.dart';
+import 'package:givt_app/features/impact_groups/repo/impact_groups_repository.dart';
+import 'package:givt_app/shared/bloc/base_state.dart';
+import 'package:givt_app/shared/bloc/common_cubit.dart';
+
+class FamilyHomeScreenCubit
+    extends CommonCubit<FamilyHomeScreenUIModel, dynamic>
+    with PreferredChurchUseCase, RegistrationUseCase, FamilySetupUseCase {
```

Comment:
You are not using these usecases so you can remove the with and the three usecases.

Reply:
copy-paste thing 😇